README.md

```diff
@@ -13,11 +13,19 @@ Puma is a **simple, fast, multi-threaded, and highly concurrent HTTP 1.1 server
 
 ## Built For Speed &amp; Concurrency
 
-Puma processes requests using a C-optimized Ragel extension (inherited from Mongrel) that provides fast, accurate HTTP 1.1 protocol parsing in a portable way. Puma then serves the request using a thread pool. Each request is served in a separate thread, so truly concurrent Ruby implementations (JRuby, Rubinius) will use all available CPU cores.
+Puma processes requests using a C-optimized Ragel extension (inherited from
```

Comment:
Did you change anything in here other than 80-char line breaks?

Reply:
I added a sentence to Quick Start, which maybe should be expanded on?  

"Puma expects to find OpenSSL development files when installed/compiled. If you want to compile it without ssl support, set ENV['DISABLE_SSL']."

I can pull this and do a separate PR.

Reply:
I removed to format changes and just added an ssl section.